fix: delete env from settings (#134)
bobheadxi/deployments: README.md

This is the same as `deactivate-env`, except deletes the environment entirely. See [`step: deactivate-env`](#step-deactivate-env) for more details.

> GITHUB_TOKEN doesn't not allow to delete environment, you have to set a personal access token and provide it as `token` option.

Reviewer: ```suggestion
Note that the default `GITHUB_TOKEN` does not allow environment deletion - you have to set a personal access token and provide it in the `token` input.
```
